### DISCUSSION TITLE
Low RAM client

### DISCUSSION THREAD

**Discussion by Junglaverde** [ltsp/ltsp#438] [Q&A]
Hello again Alkis.
A question. I have 15 computers with 4gb of ram and another 10 computers with 2gb. In the previous version, it made all 10 computers depend on the server and all 15 use its resources. In this new version how can I do it?

A greeting from Spain.
Congratulations on the work you are doing.

**Comment by alkisg** [comment]
I guess you're asking about thin clients?
If so, read https://github.com/ltsp/ltsp/discussions/243

**Reply by Junglaverde** [reply]
ok, I already read it. In my case, since all computers have at least 2GB they can work normally.

Thanks a lot. :)
